crates/bevy_render/src/mesh/mesh/mod.rs

```diff
@@ -546,6 +546,48 @@ impl Mesh {
         self
     }
 
+    /// Inverts the winding of the indices such that all counter-clockwise triangles are now
+    /// clockwise and vice versa.
+    ///
+    /// Does nothing if no [Indices] are set.
```

Comment:
```suggestion
    /// Does nothing if no [`Indices`] are set.
```

Reply:
lol that was copied from above. Will fix it there too